File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/dto/request/spot/SpotBatchOrdersReq.java
package com.bitget.openapi.dto.request.spot;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.io.Serializable;
import java.util.List;

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: spot batch order request
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class SpotBatchOrdersReq implements Serializable {

    private static final long serialVersionUID = -1L;

    /**
     * Currency pair
     */
    private String symbol;

    /**
     * order list
     */
    private List<SpotOrdersReq> orderList;
}

File: bitget-node-sdk-api/src/lib/model/spot/order/SpotBatchOrdersReq.ts
import {SpotOrdersReq} from './SpotOrdersReq';

export class SpotBatchOrdersReq {
    /**
     * Currency pair
     */
    private _symbol!:string;
    /**
     * order list
     */
    private _orderList!:SpotOrdersReq[];


    get symbol(): string {
        return this._symbol;
    }

    set symbol(value: string) {
        this._symbol = value;
    }

    get orderList(): SpotOrdersReq[] {
        return this._orderList;
    }

    set orderList(value: SpotOrdersReq[]) {
        this._orderList = value;
    }
}